tests/functional/monitoring/prometheus/metrics/test_monitoring_defaults.py

```diff
@@ -282,3 +283,43 @@ def test_provider_metrics_available(threading_lock):
         "so that the list of metrics without results is empty."
     )
     assert list_of_metrics_without_results == [], msg
+
+
+@blue_squad
+@tier1
+@bugzilla("2297285")
+@pytest.mark.polarion_id("OCS-XXX")
+def test_monitoring_ipv6(threading_lock):
```

Comment:
should we check if ipv6 feature enabled on deployment before running this test? if not we may want to skip.
we'll probably need to add marker for `ipv6_enabled`

Reply:
done `test_monitoring_ip_connectivity`